lib/services/local/users.go

```diff
@@ -602,10 +625,31 @@ func (s *IdentityService) GetWebauthnLocalAuth(ctx context.Context, user string)
 	return wal, trace.Wrap(json.Unmarshal(item.Value, wal))
 }
 
+func (s *IdentityService) GetTeleportUserByWebauthnID(ctx context.Context, webID []byte) (string, error) {
+	if len(webID) == 0 {
+		return "", trace.BadParameter("missing parameter webID")
+	}
+
+	item, err := s.Get(ctx, webauthnUserKey(webID))
+	if err != nil {
+		return "", trace.Wrap(err)
+	}
+	user := &wantypes.User{}
+	if err := json.Unmarshal(item.Value, user); err != nil {
+		return "", trace.Wrap(err)
+	}
+	return user.TeleportUser, nil
+}
+
 func webauthnLocalAuthKey(user string) []byte {
 	return backend.Key(webPrefix, usersPrefix, user, webauthnLocalAuthPrefix)
 }
 
+func webauthnUserKey(id []byte) []byte {
+	key := base64.RawStdEncoding.EncodeToString(id)
```

Comment:
nit: curious if we have any preference in general to use `base64.RawStdEncooding` vs `base64.StdEncoding`. if we gonna have more base64 in the future, may worth pick one as standard.